pkg/resources/sku/sku.go

```diff
@@ -28,7 +29,20 @@ const (
 )
 
 var (
-	defaultReplicas int32 = 1
+	products = map[v1alpha1.ProductName][]string{
```

Comment:
This is defined twice in this file. Once copy can be removed.

Reply:
moved